packages/paste-website/src/pages/components/file-picker/index.mdx

```diff
@@ -0,0 +1,259 @@
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "File Picker"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/file-picker"
+  storybookUrl="/?path=/story/components-file-picker--"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{FilePicker, FilePickerButton, Label, HelpText, useUID}} language="jsx" noInline>
+  {defaultFilePicker}
+</LivePreview>
+
+## Guidelines
+
+### About File Picker
+
+Use a File Picker when users need to upload a document in a form flow. The File Picker allows any type of file by default, but the desired file type can be specified using the <inlineCode>accept</inlineCode> prop. This is a native HTML file picker, `<input type="file" />`, so there's currently no drag-and-drop or multiple attachments functionality. <Anchor href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file" showExternal>Read more about the native file picker on MDN.</Anchor> The input is hidden visually, but still appears in the DOM, so that we're able to apply custom styles in an accessible way.
+
+### Accessibility
+
+<UnorderedList>
+  <ListItem>
+    Include a visible <Anchor href="/components/label">Label</Anchor> on <strong>all</strong> form fields.
+  </ListItem>
+  <ListItem>
+    Each label must use the <inlineCode>htmlFor</inlineCode> prop that equals the <inlineCode>id</inlineCode> of the
+    associated input.
+  </ListItem>
+  <ListItem>
+    Provide clear identification of required fields in the label or at the start of a form. If you use the required
+    field indicator, include the form key at the start of the form.
+    <UnorderedList marginBottom="space0">
+      <ListItem>
+        Use the <inlineCode>required</inlineCode> prop to programatically indicate they are required to browsers.
+      </ListItem>
+    </UnorderedList>
+  </ListItem>
+  <ListItem>
+    Include inline error text with the error icon on any field that errors to make it visually clear that the field
+    changed. You can also use{' '}
+    <Anchor href="/components/help-text#error-help-text">the "error" variant of Help Text</Anchor>.
+  </ListItem>
+  <ListItem>
+    If the File Picker has associated help text or error text, include the <inlineCode>aria-describedby</inlineCode>{' '}
+    prop on the File Picker. This should match the <inlineCode>id</inlineCode> of the help or error text.
+  </ListItem>
+</UnorderedList>
```

Comment:
suggestion: just for future reference, you don't need to use components to achieve this. Pure markdown would achieve the same thing.

```
- Include a visible [Label](/components/label) on **all** form fields.
- Each label must use the `htmlFor` prop that equals the `id` of the associated input.
```